android/app/src/debug/res/xml/react_native_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
```

Comment:
Do we really need this file in here? Or can this be ignored?

(10.0.2.2 means something entirely different on STO vs elsewhere)

Reply:
That is the default file from RN upstream. I don't know the answer to your question.

Reply:
I think the file is needed, and the localhost is needed, because I think I remember that Android 8 or 9 did or will be doing forced-HTTPS suggestions like iOS does? (If that's already a thing, then IDK.)